docs/find-options.md

```diff
@@ -23,7 +23,7 @@ userRepository.find({ relations: ["profile", "photos", "videos", "videos.video_a
 * `join` - joins needs to be performed for the entity. Extended version of "relations".
 
 ```typescript
-userRepository.find({ 
+userRepository.find({
```

Comment:
This change shouldn' t be added. The same goes for others in this PR

Reply:
You can remove what you  see isn't necessary